www/index.php

```diff
@@ -29,8 +29,8 @@
 translations have separate mailing lists.</p>
 
 <h2>Latest News!</h2>
-<p>On March 2014 site was almost totally rebuilded. Now it uses same design as
-new php.net pages. Many of tools and features has been removed, because they weren't used.
+<p>In March 2014 site was almost totally rebuilt. Now it uses same design as
```

Comment:
"On March 2014" is correct. Other wording fixes are correct. An additional improvement is changing the wording for "Many of tools and features" to "Many of the tools and features".